<launch>

  <arg name="yaml" default="azure_robotarm_seg"/>

  <rosparam command="load" file="$(find assembly_part_recognition)/param/azure_robotarm_seg.yaml" ns="azure_robotarm_seg" />
  <node name="robotarm_segmenter" pkg="assembly_part_recognition" type="robotarm_segmenter.py" output="screen"/>
</launch>